frontend/src/components/common/form/InputGroup.tsx

```diff
@@ -116,14 +116,32 @@ export const InputGroup: React.FC<InputGroupProps> = ({
             {...rest}
           />
         )}
-      </div>
+      </ColContent>
       {postText && (
         <BootstrapInputGroup.Append>
           <BootstrapInputGroup.Text className={disabled ? 'append-disabled' : ''}>
             {postText}
           </BootstrapInputGroup.Text>
         </BootstrapInputGroup.Append>
       )}
-    </div>
+    </Row>
   );
 };
+
+const ColPrepend = styled(Col)`
+  padding-right: 0;
+  .form-control {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+  }
+`;
+
+const ColContent = styled(Col)`
+  &:not(:first-child) {
+    padding-left: 0;
+    .form-control {
+      border-top-left-radius: 0;
+      border-bottom-left-radius: 0;
+    }
+  }
+`;
```

Comment:
question - this is a potentially significant change given that all forms use this component. Did you run a pass on all forms to check nothing got broken?

I'd personally hold off on making these changes (unless no risk of breaking existing layouts) until we get our CSS "building blocks" from Bobbi

Reply:
So far it looks like is only used by the PropertyInformation and PropertyFilterOptions components, this change is pertinent to the later one. I checked the details tab and it looks good to me

Reply:
The number of updated snapshots indicate otherwise. If you are sure it doesn't break existing UI I'm fine with it.

Reply:
Right, that sounds like a bad CSS idea... Ill take a double check on the ones that might have used it

Reply:
I checked and it looks OK in my local deployment. Thoughts on checking on dev? I add the deleted file if the layout is broken